app/src/main/res/values-fr/strings.xml

```diff
@@ -181,13 +181,13 @@
     <string name="popup_tags_title">Étiquettes disponibles</string>
     <string name="popup_tags_actions">Actions disponibles</string>
     <string name="show_history">Afficher l\'historique</string>
-    <string name="unable_to_initialize_shortcuts">Impossible d’initialiser les raccourcis. KISS est-il votre lanceur par défaut \?</string>
+    <string name="unable_to_initialize_shortcuts">Impossible d\'initialiser les raccourcis. KISS est-il votre lanceur par défaut \?</string>
     <string name="pref_tags_menu_dismiss">Fermer le menu des étiquettes après sélection</string>
-    <string name="show_untagged">Afficher non étiquetté</string>
-    <string name="pref_show_untagged">Afficher action non étiquettée</string>
+    <string name="show_untagged">Afficher non étiqueté</string>
+    <string name="pref_show_untagged">Afficher actions non étiquetées</string>
     <string name="reset_excluded_from_history_apps_name">Réinitialiser la liste des applications exclues de l\'historique</string>
     <string name="reset_excluded_from_history_apps_warn">Êtes-vous sûr·e de vouloir réinitialiser la liste des applications exclues de l\'historique \?</string>
-    <string name="settings_excluded_apps">Élements exclues</string>
+    <string name="settings_excluded_apps">Éléments exclus</string>
```

Comment:
"élément" is masculine. Fixed the missing accent on the second "e".
https://www.dictionnaire-academie.fr/article/A9E0755